src/main/java/uk/gov/register/serialization/CommandParser.java

```diff
@@ -30,7 +39,10 @@ public RegisterCommand newCommand(String s){
             case "add-item":
                 if (parts.length == 2) {
                     try {
-                        Item item = new Item(new ObjectMapper().readTree(parts[1]));
+                        String jsonContent = parts[1];
+                        canonicalJsonValidator.validateItemStringIsCanonicalized(jsonContent);
+                        String itemHash = DigestUtils.sha256Hex(jsonContent.getBytes());
```

Comment:
Can we use `getBytes(StandardCharsets.UTF_8)`? This would be consistent with the what currently happens in `ObjectReconstructor` for the original /load endpoint.